uaa/src/test/java/org/cloudfoundry/identity/uaa/degraded/DegradedSamlLoginTests.java

```diff
@@ -61,23 +60,45 @@
     @Rule
     public ScreenshotOnFail screenShootRule = new ScreenshotOnFail();
 
+    @Value("${PUBLISHED_HOST:predix-uaa-integration}")
+    String publishedHost;
+
+    @Value("${CF_DOMAIN:run.aws-usw02-dev.ice.predix.io}")
+    String cfDomain;
+
+    @Value("${BASIC_AUTH_CLIENT_ID:app}")
+    String basicAuthClientId;
+
+    @Value("${BASIC_AUTH_CLIENT_SECRET:appclientsecret}")
+    String basicAuthClientSecret;
+
     @Autowired
     RestOperations restOperations;
 
     @Autowired
+    public Environment environment;
+
+    @Autowired
     WebDriver webDriver;
 
     protected final static Logger logger = LoggerFactory.getLogger(DegradedSamlLoginTests.class);
     private final static String zoneSubdomain = "test-app-zone";
+    private String baseUaaZoneHost;
+    private String protocol;
     private String baseUrl;
+    private String testRedirectUri;
     private String zoneAdminToken;
+    private String baseBasicAuth;
 
 
     @Before
     public void setup() throws Exception {
-        baseUrl = "http://" + zoneSubdomain + ".localhost:8080/uaa";
+        System.out.println("Environment"+environment);
+        baseUaaZoneHost = environment.getProperty("runAgainstCloud") != null ? (publishedHost + "." + cfDomain) : "localhost:8080/uaa";
```

Comment:
The variable name "runAgainstCloud". Same as the other one... RUN_AGAINST_CLOUD.